extension/app/ts/components/simulationExplaining/identifyTransaction.tsx

```diff
@@ -140,7 +141,7 @@ export function identifyTransaction(simTx: SimulatedAndVisualizedTransaction): I
 		signingAction: 'Transfer Ether',
 		simulationAction: 'Simulate Ether Transfer',
 		rejectAction: 'Reject Ether Transfer',
-		identifiedTransaction: simTx,
+		identifiedTransaction: { ...simTx, to: simTx.transaction.to as AddressBookEntry }
```

Comment:
we should not be doing casting here

Reply:
Removed, found that other tx types expect ``simTx.to`` to exist when the `to` is not undefined. ``simTx.to`` was never set anywhere during `formSimulatedAndVisualizedTransaction` so we now set it.